moveit_ros/perception/mesh_filter/test/mesh_filter_test.cpp

```diff
@@ -231,7 +231,7 @@ void MeshFilterTest<Type>::test()
     float sensor_depth = sensor_data_[idx] * FilterTraits<Type>::ToMetricScale;
     if (fabs(sensor_depth - distance_ - shadow_) > epsilon_ && fabs(sensor_depth - distance_) > epsilon_)
     {
-      ASSERT_NEAR(filtered_depth[idx], gt_depth[idx], 1e-6);
+      ASSERT_NEAR(filtered_depth[idx], gt_depth[idx], 1e-4);
```

Comment:
The error you gave [in this post](https://github.com/ros-planning/moveit/issues/2066#issuecomment-653302801)
had 0 values in one of the entries. How can this be solved by reducing the accuracy by factor 1e2?

Reply:
These are unrelated issues. The post you are referring to is related to Melodic. This one is related to Noetic, which utilizes another version of Mesa OpenGL.